getting_started/install_openshift.adoc

```diff
@@ -30,6 +30,7 @@ cluster must be IBM POWER servers.
 * Wildcard DNS resolution that resolves your domain to the IP of the node. So,
 an entry like the following in your DNS server:
 +
+[source,terminal]
```

Comment:
Just checking for this and the below code block, that terminal is actually correct as the source? It looks like it's something you fill in somewhere in the DNS config, not a command or output in the terminal.

Reply:
Oop, I think you're right. Good catch!